package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ui-kit",
   "private": true,
   "scripts": {
-    "setup": "npm i && lerna bootstrap",
+    "setup": "npm i && husky install && lerna bootstrap",
```

Comment:
hmm hook problems?

Reply:
When initializing husky, it now adds `"prepare": "husky install"` to the `scripts` in `package.json`, so I figured this belonged here instead.